snapshots/government-organisations-independent-reconfiguration-panel-about-membership.html

```diff
@@ -0,0 +1,255 @@
+</p>        <h1 class="main">Membership</h1>
+        <p class="description">
+          Information about the Independent Reconfiguration Panel&#x2019;s members, meeting minutes and business reviews.
+        </p>
+        <div class="body">
+          <div class="govspeak">
+<h2 id="members">Members</h2>
+
+<p>The Independent Reconfiguration Panel (<abbr title="Independent Reconfiguration Panel">IRP</abbr>) is chaired by Lord Bernard Ribeiro.</p>
+
+<p>Its members are:</p>
+
+<ul>
+  <li>Cath Broderick </li>
+  <li>Fiona Campbell </li>
+  <li>Shera Chok, Director of Primary Care at Barts Health NHS Trust </li>
+  <li>Nick Coleman, Associate Medical Director at the University Hospital of North Staffordshire  </li>
+  <li>Glenn Douglas, Chief Executive of the Maidstone and Tunbridge Wells NHS Trust  </li>
+  <li>Shane Duffy  </li>
+  <li>Rosemary Granger </li>
+  <li>Tessa Green </li>
+  <li>Jane Hawdon, Clinical Academic Group Director (Children&#x2019;s Health) at Barts Health NHS Trust  </li>
+  <li>Nicky Hayes </li>
+  <li>Brenda Howard  </li>
+  <li>Simon Morritt, Chief Executive of Sheffield Children&#x2019;s NHS Foundation Trust   </li>
+  <li>Linn Phipps</li>
+  <li>Hugh Ross   </li>
+  <li>Suzanne Shale</li>
+</ul>
+
+<h2 id="minutes-of-irp-meetings">Minutes of <abbr title="Independent Reconfiguration Panel">IRP</abbr> meetings</h2>
+
+<p><a href="https://www-origin.integration.publishing.service.gov.uk/government/organisations/independent-reconfiguration-panel/about">9 July 2015</a> (ODF, 67KB)    <br/>
+<a href="https://www-origin.integration.publishing.service.gov.uk/government/organisations/independent-reconfiguration-panel/about">14 May 2015</a> (ODF, 66.5KB) <br/>
+<a href="https://www-origin.integration.publishing.service.gov.uk/government/organisations/independent-reconfiguration-panel/about">8 January 2015</a> (ODF, 63.5KB)     <br/>
+<a href="https://www-origin.integration.publishing.service.gov.uk/government/organisations/independent-reconfiguration-panel/about">13 November 2014</a> (ODF, 62KB)    <br/>
+<a href="https://www-origin.integration.publishing.service.gov.uk/government/organisations/independent-reconfiguration-panel/about">10 July 2014</a> (ODF, 54.5KB)    <br/>
+<a href="https://www-origin.integration.publishing.service.gov.uk/government/organisations/independent-reconfiguration-panel/about">8 May 2014</a> (ODF, 69KB)    <br/>
+<a href="https://www-origin.integration.publishing.service.gov.uk/government/organisations/independent-reconfiguration-panel/about">14 November 2013</a> (ODF, 59.5KB)    <br/>
+<a href="https://www-origin.integration.publishing.service.gov.uk/government/organisations/independent-reconfiguration-panel/about">12 September 2013</a> (ODF, 55KB)    <br/>
+<a href="https://www-origin.integration.publishing.service.gov.uk/government/organisations/independent-reconfiguration-panel/about">11 July 2013</a> (ODF, 30.5KB)      </p>
+
```

Comment:
All these links go to HTML page on integration, so we need the production URLs and to remove the file type/size at the end